README.rst

```diff
@@ -34,69 +34,91 @@ If your version of python3 is too old an error of this type or similar will be e
     ERROR: Could not find a version that satisfies the requirement rkvst-simplehash (from versions: none)
     ERROR: No matching distribution found for rkvst-simplehash
 
-Example
+Examples
 =============
 
 You can then use the code to recreate the simple hash of a list of SIMPLE_HASH events from RKVST.
 
+Importing in own code
+.....................
+
 .. code:: python
 
     """From a list of events.
     """
 
-    from rkvst_simplehash.v1 import hash_events
+    from rkvst_simplehash.v1 import (
+        anchor_events,
+        SimpleHashError,
+    )
+
+    with open("credentials/token", mode="r", encoding="utf-8") as tokenfile:
+        auth_token = tokenfile.read().strip()
 
+    # SimpleHashClientAuthError is raised if the auth token is invalid or expired.
     # if any pending events a SimpleHashPendingEventFound error will be thrown
     # if any of the events do not contain the required field then a SimpleHashFieldMissing error will be thrown
+    try:
+        simplehash = anchor_events_events(
+            "2022-10-07 07:01:34Z",
+            "2022-10-16T13:14:56Z",
+            "app.rkvst.io",
+            auth_token,
+        )
+    except SimpleHashError as ex:
+        print("Error", ex)
 
-    simplehash = hash_events(events)
+    else:
+        print("simplehash=", simplehash)
 
 
 Command Line
 ------------
 
-This functionality is also available on the cmdline. Execute the simplehash query using curl and pip to the
-rkvst_simplehashv1 command or the v1.py script.
-
-Installing locally:
-....................
-
-Execute this after cloning the repo.
-
-.. code:: bash
-
-    python3 -m pip install bencode.py~=4.0
-
-    URL="https://app.rkvst.io/archivist/v2/assets/-/events"
-    SINCE=timestamp_accepted_since=2022-10-19T00:00:00Z
-    BEFORE=timestamp_accepted_before=2022-10-26T00:00:00Z
-    BEARER_TOKEN_FILE=credentials/token
-
-    curl -s -X GET \
-         -H "@$BEARER_TOKEN_FILE" \
-         "$URL?proof_mechanism=SIMPLE_HASH&${SINCE}&${BEFORE}" | rkvst_simplehash/v1.py
-
+This functionality is also available on the cmdline.
 
 Using a virtual env and published wheel:
 ........................................
 
 This can be executed anywhere using a virtualenv and published wheel.
+Credentials are stored in files in credentials directory.
+
+Using an auth token directly:
 
 .. code:: bash
 
+    #!/usr/bin/env bash
+    #
     python3 -m venv simplehash-venv
     source simplehash-venv/bin/activate
-    python3 -m pip install rkvst_simplehash
+    python3 -m pip install -q rkvst_simplehash
+    
+    rkvst_simplehashv1 \
+        --auth-token-file "credentials/token" \
+        --fqdn "app.rkvst.io" \
```

Comment:
lets remove fqdn, we are including it to enable testing? and set the default to be app.rkvst.io